pages/common/openscad.md

```diff
@@ -0,0 +1,16 @@
+# openscad
+
+> Software for creating solid 3D CAD objects.
+> More information: <https://openscad.org>.
+
+- Open a file:
```

Comment:
Can we open only a file with `scad` extension or anything else as well ?

Reply:
OpenSCAD only works with `scad` files, something like `openscad x.stl` opens up creating a new `scad` like `import("/path/to/x.stl");`